gc/base/HeapVirtualMemory.cpp

```diff
@@ -312,7 +312,19 @@ MM_HeapVirtualMemory::commitMemory(void* address, uintptr_t size)
 {
 	MM_GCExtensionsBase* extensions = MM_GCExtensionsBase::getExtensions(_omrVM);
 	MM_MemoryManager* memoryManager = extensions->memoryManager;
-	return memoryManager->commitMemory(&_vmemHandle, address, size);
+
+	bool resultCommitMemory = memoryManager->commitMemory(&_vmemHandle, address, size);
+
+	if (resultCommitMemory && extensions->pretouchHeapOnExpand) {
+		uintptr_t pageSize = _vmemHandle.getVirtualMemory()->getPageSize();
```

Comment:
This class has implementation of `MM_HeapVirtualMemory::getPageSize()` already you can use here.